dscreator/sources/ferrybox/extractor.py

```diff
@@ -21,10 +21,15 @@ class TrajectoryExtractor(BaseExtractor):
     variable_codes: List[str]
     variable_uuid_map: dict[str, str]
     qc_flags: List[int]
+    with_qc: bool = True
     qc_variables: List[str] = field(init=False)
 
     def __post_init__(self):
-        self.qc_variables = [f"{v}_qc" for v in self.variable_codes]
+        self.variable_uuid_map = {k: v for k, v in self.variable_uuid_map.items() if k in self.variable_codes + ["track"]}
+        if self.with_qc:
```

Comment:
What will these variables be? As the only reflectance data we keep in tsb are "good", no?

Reply:
It is if we use the qc column, but in most cases it is "good" yep